src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -298,12 +298,12 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
                           ? html`
                               ${formatTime(
                                 new Date(item.datetime),
-                                this.hass!.language
+                                this.hass!.locale
                               )}
                             `
                           : html`
                               ${new Date(item.datetime).toLocaleDateString(
```

Comment:
We should use `formatDate` here

Reply:
`formatDate` does not support the options needed here to format the date as just the short weekday. I can submit a separate PR to add that capability and go through the codebase to use it where appropriate after this PR is merged.